gameboy_opengl/src/lib.rs

```diff
@@ -51,12 +51,11 @@ pub fn start(rom: Vec<u8>) -> Result<(), String> {
     canvas.set_draw_color(Color::RGB(0, 0, 0));
     canvas.clear();
 
-    let mut cartridge = Cartridge::from_rom(rom);
-    load_ram_save_data(&mut cartridge).map_err(|e| format!("{:?}", e))?;
-    load_timestamp_data(&mut cartridge).map_err(|e| format!("{:?}", e))?;
-
     let rtc = Box::new(NativeRTC::new());
-    let mut emulator = Emulator::from_cartridge(cartridge, rtc);
+    let mut emulator = Gameboy::from_rom(rom, rtc);
+
+    load_ram_save_data(emulator.get_cartridge_mut());
```

Comment:
We don't want to loose error information by ignoring the `Result` returned by the `load` methods. Let's add back the `.map_err(|e| format!("{:?}", e))?;` to these function calls.

Reply:
I added in the map_error methods to the load functions. Must have missed the change when resolving merge conflicts.